added demo splash screen

1zun4secondary opened the pull request.

1zun4 committed:
added demo splash screen

1zuna-chan commented:
Don't merge yet. It would break nightly builds! Minecraft has to be remapped on the JS engine.

1zun4 committed:
fixed splashscreen

1zun4 committed:
# Conflicts:
#	src/main/kotlin/net/ccbluex/liquidbounce/render/ultralight/UltralightEngine.kt

1zun4 committed:
renamed undefined view to splash view

1zun4 committed:
fixed basic splash screen

1zun4 committed:
Merge branch 'nextgen' into splashscreen_demo

# Conflicts:
#	src/main/kotlin/net/ccbluex/liquidbounce/render/ultralight/UltralightEngine.kt
